src/main/java/gim/model/exercise/Exercise.java

```diff
@@ -125,4 +125,12 @@ public String toString() {
         return builder.toString();
     }
 
+    @Override
+    public int compareTo(Exercise exercise) {
```

Comment:
For the `compareTo` method here, do we need to also consider the equality of values for the other fields such as `Weight`, `Sets` and `Reps` as well? What would be a better definition of equality between two `Exercise` objects?

Reply:
I would say the `Weight`, `Sets` and `Reps` are not part of the comparison. The primary comparison between two `Exercise` objects should be based on `Date`. Should the two exercises have the same `Date`, they will then be compared lexicographically! Do correct me if I am wrong! @ErvinK123 @kavantan @ee-suan

Reply:
I see, that sounds great! Will make the necessary changes.